ph5/utilities/ph5validate.py

```diff
@@ -847,14 +831,32 @@ def get_args():
 
     args = parser.parse_args()
 
-    # Add formatter
-    args.level = str(args.level).upper()
-    if args.level != "ERROR" and args.level != "WARNING" and \
-            args.level != "INFO":
-        raise ValueError("Invalid logging level.")
+    outfile = 'ph5_validate.log' if args.outfile is None else args.outfile
+    PH5 = args.nickname
+    if not os.path.exists(PH5) and not os.path.exists(PH5 + '.ph5'):
+        LOGGER.error("{0} not found.".format(PH5))
+        sys.exit()
+    else:
+        # Set up logging
+        # Write log to file
+        ch = logging.FileHandler(os.path.join('.', outfile))
+        ch.setLevel(logging.INFO)
+        if args.verbose is True:
+            ch.setLevel(logging.DEBUG)
+        # Add formatter
+        formatter = logging.Formatter(LOGGING_FORMAT)
+        ch.setFormatter(formatter)
+        LOGGER.addHandler(ch)
+        level = args.level.upper()
+        if level == "ERROR":
+            LOGGER.setLevel(logging.ERROR)
+        elif level == "WARNING":
+            LOGGER.setLevel(logging.WARNING)
+        elif level == "INFO":
+            LOGGER.setLevel(logging.INFO)
+        else:
+            raise PH5ValidateException("Invalid Logging Level %s" % level)
```

Comment:
This is messy.
And does verbose just set log level to debug?

Reply:
why `LOGGER.setLevel()` and `ch.setLevel()`?

Reply:
Yes, I see this is messy too. I wasn't sure about this part. I think I will change this part to use ch.setLevel() and verbose is the first priority before level.